docs/install/registries.md

```diff
@@ -25,6 +25,92 @@ To configure a private registry scoped to a particular organization:
 "@myorg3" = { token = "$npm_token", url = "https://registry.myorg.com/" }
 ```
 
+### Changing the Mirror Source in Bun
+
+To change the mirror source in Bun, you can configure the `bunfig.toml` file. Here are the steps:
+
+1. Open the `bunfig.toml` file in your project directory. If it doesn't exist, create a new file named `bunfig.toml`.
+
+2. Add the following configuration to set the mirror source:
+
+```toml
+[install]
+# set the mirror source URL
+registry = "https://your-mirror-source-url"
+```
+
+3. If your mirror source requires authentication, you can include the token or username/password:
+
+```toml
+[install]
+# set the mirror source URL with token
+registry = { url = "https://your-mirror-source-url", token = "your-token" }
+
+# set the mirror source URL with username/password
+registry = "https://username:password@your-mirror-source-url"
+```
+
+4. Save the `bunfig.toml` file.
+
+For more detailed information on configuring the mirror source and other registry settings, refer to the [custom registry guide](../guides/install/custom-registry.md).
+
+### Available Mirror Registry
```

Comment:
I don't see a reason why we should maintain a list of available npm mirrors. We're recommending them but have no guarantee that they are/will be OK.

Reply:
So maybe we can just update the word of the title to `Recommended Mirror Registry` ?

Reply:
I don't think that will solve the issue. We shouldn't provide any mirror where we can't guarantee safe operation.

Reply:
We should add recommend mirror registery. Or this PR is invalid